spec/kong_tests.conf

```diff
@@ -4,7 +4,6 @@ proxy_listen = 0.0.0.0:9000
 proxy_listen_ssl = 0.0.0.0:9443
 cluster_listen = 0.0.0.0:9946
 cluster_listen_rpc = 127.0.0.1:9373
-ssl = on
```

Comment:
My thoughts on this:

* I could see this being applicable in some scenarios, but it would break the OAuth 2.0 and SSL plugin. As a user I would rather block it with my firewall/security groups, but yeah I can see that somebody may want this.

* By default it would be `on`, so not to disrupt the plugins. Because of this the SSL certificates would still need to be auto-generated (the user can always change them).

* Not really requested by anybody so far. I would focus the `refactor/cli` purely on the refactoring aspect, without changing/adding features.

I will add it back anyways..but we should evaluate new features, especially if they break plugins. (Adding an SSL or OAuth 2.0 plugin when `ssl = false` could trigger an error/warning, for example).